File: Python praktikum/prak3/soal2.py
tarif = 10000
cuaca = str(input("Saat perjalanan hujan atau terang = ")) 
jarak = int(input("Berapa KM jarak yang ditempuh = "))
waktu = int(input("Berapa Jam Waktu yang ditempuh = "))

if(jarak > 3):
    tarif = int(tarif + ((jarak - 3) * 2000))
else :
    tarif = tarif

if(waktu > 2 * jarak):
    tarif = int(tarif + ((waktu - (2* jarak)) * 1000))
else :
    tarif = tarif

if(cuaca == "hujan"):
    tarif = int(tarif * 115 / 100)
else :
    tarif = tarif

print("Tarif = ", tarif)
    

File: java praktikum/prak10/src/praktikum/pkg10/Soal2.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package praktikum.pkg10;

import java.util.Scanner;

/**
 *
 * @author Nrhys
 */
public class Soal2 {
    public static void main(String[] args) {
        int jarak, waktu, tarif; 
        String cuaca;
        Scanner input = new Scanner(System.in);
        System.out.println("Inputkan Jarak Tempuh (dalam km) = ");
        jarak = input.nextInt();
        System.out.println("Inputkan Waktu Tempuh = ");
        waktu = input.nextInt();
        System.out.println("Inputkan Cuaca = ");
        cuaca = input.next();
          
        
        tarif = 10000;
        
        if(jarak > 3){
            tarif = tarif + ((jarak - 3) * 2000);
        }else{
            
        }
        
        if(waktu > 2 * jarak){
            tarif = tarif + ((waktu - (2* jarak)) * 1000);
        }else{
            
        }
        
        if("hujan".equals(cuaca)){
            tarif = tarif * 115 / 100;
        }else{
            
        }
        System.out.println("Tarif = " + tarif);
    }
}
